providers/n2kAnalyzer.js

```diff
@@ -36,7 +36,9 @@ function N2KAnalyzer() {
   var that = this;
   this.linereader.on('line', function(data) {
     try {
-      that.push(JSON.parse(data));
+      parsed = JSON.parse(data)
+      that.push(parsed);
+      options.app.emit("n2KAnalyzerOut", parsed)
```

Comment:
Let's make this `N2KAnalyzerOut`, if for nothing but consistence with the class name.